### PR TITLE
LEAF-1462: Added min width to dialog box

### PR DESCRIPTION
**Pull request by abiola-adewuyi**
Implemented feedback from Michael. Dialog box cannot be minimized less than the width of the dialog box content.

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5441467&bid=17844215)

File: LEAF_Nexus/js/dialogController.js [hunk 1]
```diff
@@ -18,10 +18,15 @@ function dialogController(containerID, contentID, indicatorID, btnSaveID, btnCan
...
 
+	//calculate min width of dialog based on min width of content div
+	var minWidth = parseInt($('#' + this.contentID).css('min-width'));
+	minWidth = (minWidth == 0) ? 0 : (minWidth + 30);
+
 	$('#' + this.containerID).dialog({autoOpen: false,
 										modal: true,
 										height: 'auto',
-										width: 'auto'});
+										width: 'auto',
```
Comment (dastan-rahimi) [inline LEAF_Nexus/js/dialogController.js:28]: Fix indentation

File: LEAF_Request_Portal/js/dialogController.js [hunk 2]
```diff
@@ -20,10 +20,15 @@ function dialogController(containerID, contentID, indicatorID, btnSaveID, btnCan
...
 	this.incomplete = 0;
 
+	//calculate min width of dialog based on min width of content div
+	var minWidth = parseInt($('#' + this.contentID).css('min-width'));
+	minWidth = (minWidth == 0) ? 0 : (minWidth + 30);
+
 	$('#' + this.containerID).dialog({autoOpen: false,
 										modal: true,
 										height: 'auto',
-										width: 'auto'});
```
Comment (dastan-rahimi) [inline LEAF_Request_Portal/js/dialogController.js:26]: fix indentation

File: LEAF_Nexus/js/dialogController.js (hunk 1 above)
Comment (abiola-adewuyi) [reply LEAF_Nexus/js/dialogController.js:28]: The indentation is fine in the file itself locally.

File: LEAF_Request_Portal/js/dialogController.js (hunk 2 above)
Comment (abiola-adewuyi) [reply LEAF_Request_Portal/js/dialogController.js:26]: The indentation is fine in the file itself locally.

File: LEAF_Nexus/js/dialogController.js (hunk 1 above)
Comment (thomasnguyen95) [reply LEAF_Nexus/js/dialogController.js:28]: Please replace tab spacing with 4 spaces

**Comment by thomasnguyen95** [comment]
> Codacy Here is an overview of what got changed by this pull request:
> 
> ```diff
> Issues
> ======
> - Added 4
>            
> ```
> 
> See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5441467&bid=17844215)

Please review the codacy checks.

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5441467&bid=17844215)

**Pull request merged**